lib/Sabre/VObject/Property.php

```diff
@@ -195,6 +195,15 @@ public function serialize() {
             '\n',
             '',
         );
+
+        // avoid double-escaping of \, and \; from Compound properties
+        if (method_exists($this, 'setParts')) {
```

Comment:
Would not be better to filter by instance names instead of the method existence?